compose/src/main/java/com/bottlerocketstudios/compose/resources/Colors.kt

```diff
@@ -10,19 +11,19 @@ val br_light_red = Color(0xffe66963)
 val brown_grey = Color(0xff979797)
 val greyish_brown = Color(0xff515151)
 val white = Color(0xffffffff)
-val background_black = Color(0x05000000)
+val mostly_white = Color(0xfff8f8f8)
 val black = Color(0xff000000)
 val sea_foam = Color(0xff1fada8)
-val brown_grey_two = Color(0xff797979)
-val brown_grey_three = Color(0xff9e9e9e)
+// val brown_grey_two = Color(0xff797979)
+// val brown_grey_three = Color(0xff9e9e9e)
```

Comment:
@BottleRocket-Colin Are these colors used in the zeplin project comps? Do we still need them? If not, remove them. If so, why comment them out?

Reply:
God point... I'm deleting with extreme action!